src/NzbDrone.Core/Indexers/RssParser.cs

```diff
@@ -257,11 +257,12 @@ protected virtual RssEnclosure[] GetEnclosures(XElement item)
                                  {
                                      try
                                      {
+                                         var lengthValue = v.Attribute("length")?.Value;
                                          return new RssEnclosure
                                                 {
-                                                    Url = v.Attribute("url").Value,
-                                                    Type = v.Attribute("type").Value,
-                                                    Length = (long)v.Attribute("length")
+                                                    Url = v.Attribute("url")?.Value,
+                                                    Type = v.Attribute("type")?.Value,
+                                                    Length = string.IsNullOrEmpty(lengthValue) ? default(long) : XmlConvert.ToInt64(lengthValue)
```

Comment:
We have an extension that makes this cleaner:

`Length = v.Attribute("length")?.Value.ParseInt64() ?? 0`;

No need for the separate `lengthValue` variable and `0` is more obvious than `default(long)` I think.

Reply:
thanks! solved. default is beneficial compared to 0 because you can search for references with default. (should you ever have the question "What are the places where I'm setting the default value?". I simplified it to the literal expression instead of using the operator, hopefully that makes it easier to read.